dist/linux/README.md

```diff
@@ -9,16 +9,30 @@ sudo make install
 
 ## Systemd Service (optional)
 
-Copy and enable the service:
+You can configure the program to run as systemd service, user- or system-wide (only one at a time).
+
+### As user service:
 
 ```sh
-cp byedpi.service ~/.config/systemd/user/
-systemctl --user daemon-reload
-systemctl --user enable byedpi.service
-systemctl --user start byedpi.service
+cp byedpi_user.service ~/.config/systemd/user/byedpi.service
+cp byedpi_options ~/.config/byedpi
+systemctl --user enable --now byedpi.service
 ```
 
 You should see the service now marked as "active":
 ```sh
 systemctl --user status byedpi.service
 ```
+
+### As system service:
+
+```sh
+sudo cp byedpi_system.service /usr/lib/systemd/system/byedpi.service
+sudo cp byedpi_options /etc/byedpi
```

Comment:
I think `/etc/default/` path would be better because this is not a config but env file. 
This is controversial though.

Reply:
I wanted to avoid the need for sudo rights.

Reply:
I don't have a strong opinion on this matter.

Reply:
Other possible paths: `/etc/sysconfig`, `~/.config/`
Look at `grep -R EnvironmentFile /etc/systemd/ /usr/lib/systemd/`.

Reply:
`/etc/conf.d` is probably also a good option. I have seen it used by many openrc services for configuration.

Reply:
Looked at the files that grep found on my computer. In them, the developers/maintainers did not bother much with either the location or even the naming. So I left the settings in `etc/byedpi`.

Reply:
Both `man hier` and `man file-hierarchy` say nothing for configuration of services :(

Reply:
By the way, it might make sense to add `.conf` to the config file. However, I have only seen such naming in `EnvironmentFile=-/etc/locale.conf`.

Reply:
Yes, i think using `byedpi.conf` probably will be more readable in both repository and filesystem instead of `byedpi_options`.